src/ansys/systemcoupling/core/adaptor/impl/injected_commands.py

```diff
@@ -106,6 +106,9 @@ def _wrap_add_participant(
 
         return part_mgr.add_participant(participant_session=session.system_coupling)
 
+    if input_file := kwargs.get("input_file", None):
+        part_mgr.upload_file(input_file)
+
```

Comment:
@ochernuk when I discussed this with Seun, I mentioned that we had become aware of limitations with upload and download, in that no directory components are allowed - everything just goes to/from the top level working directory in the container. I wondered whether we should have any additional checks for this of the `input_file` parameter. I guess an invalid filename will give an error anyway, but the message might not make it obvious what the issue is.

Reply:
@iboyd-ansys I've added the check we discussed in my latest commit, please review and let me know if you have any more suggestions on how to do this.